.github/workflows/trigger_ab_tests.yml

```diff
@@ -0,0 +1,23 @@
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  trigger_ab_test:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.forced == false }}
+    steps:
+      - name: "Trigger Buildkite Pipeline"
+        run: |
+          curl -X POST https://api.buildkite.com/v2/organizations/firecracker/pipelines/performance-a-b-tests/builds \
+               -H 'Content-Type: application/json' \
+               -H 'Authorization: Bearer ${{ secrets.BUILDKITE_TOKEN }}' \
+               -d '{
+                    "commit": "HEAD",
```

Comment:
Would it be useful to pass the specific SHA here? Can we see on buildkite the specific commit each test run relates to?

Reply:
Buildkite resolves this for us, see https://buildkite.com/firecracker/roypat-collect-cascake-lake-baselines/builds/531#018abcfe-6c06-49f4-9db8-32a18a210df7

Reply:
Cool.